autopush-common/src/db/mod.rs

```diff
@@ -199,12 +222,40 @@ impl DynamoStorage {
         channel_id: &Uuid,
         message_month: &str,
         endpoint: &str,
+        register_user: Option<&DynamoDbUser>,
```

Comment:
I wonder if we should call this `deferred_user`. If so, we can if someone is aesthetically inclined.